pkgs/development/tools/ameba/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "ameba";
-  version = "0.10.1";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
     owner  = "crystal-ameba";
     repo   = "ameba";
-    rev    = "v0.10.1";
-    sha256 = "0dcw7px7g0c5pxpdlirhirqzhcc7gdwdfiwb9kgm4x1k74ghjgxq";
+    rev    = "v0.11.0";
```

would you mind adding `rec` to `stdenv.mkDerivation`, and use version here?
```suggestion
    rev    = "v${version}";
```